src/plugins/auto_batch/tests/functional/behavior/ov_plugin/auto_batching_tests.cpp

```diff
@@ -31,6 +31,42 @@ INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test,
                                             ::testing::ValuesIn(num_batch)),
                          AutoBatching_Test_DetectionOutput::getTestCaseName);
 
+INSTANTIATE_TEST_SUITE_P(nightly_CPU_AutoBatching,
+                         AutoBatching_Test,
+                         ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_CPU),
+                                            ::testing::ValuesIn(get_vs_set),
+                                            ::testing::ValuesIn(num_streams),
+                                            ::testing::ValuesIn(num_requests),
+                                            ::testing::ValuesIn(num_batch)),
+                         AutoBatching_Test::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(nightly_CPU_AutoBatching,
+                         AutoBatching_Test_DetectionOutput,
+                         ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_CPU),
+                                            ::testing::ValuesIn(get_vs_set),
+                                            ::testing::ValuesIn(num_streams),
+                                            ::testing::ValuesIn(num_requests),
+                                            ::testing::ValuesIn(num_batch)),
+                         AutoBatching_Test_DetectionOutput::getTestCaseName);
+
+// INSTANTIATE_TEST_SUITE_P(nightly_GPU_AutoBatching,
+//                          AutoBatching_Test,
+//                          ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_GPU),
+//                                             ::testing::ValuesIn(get_vs_set),
+//                                             ::testing::ValuesIn(num_streams),
+//                                             ::testing::ValuesIn(num_requests),
+//                                             ::testing::ValuesIn(num_batch)),
+//                          AutoBatching_Test::getTestCaseName);
+
+// INSTANTIATE_TEST_SUITE_P(nightly_GPU_AutoBatching,
+//                          AutoBatching_Test_DetectionOutput,
+//                          ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_GPU),
+//                                             ::testing::ValuesIn(get_vs_set),
+//                                             ::testing::ValuesIn(num_streams),
+//                                             ::testing::ValuesIn(num_requests),
+//                                             ::testing::ValuesIn(num_batch)),
+//                          AutoBatching_Test_DetectionOutput::getTestCaseName);
```

Comment:
@ilya-lavrenov I moved the Batch with GPU test from GPU Plugin test to Batch Plugin & labeled with nightly_, expected this tests will not run in pre-commit CI test, but run in nightly test. But in this CI test, those tests run & failed, do u know where the control script is? I add add the filter --gtest_filter="*smoke*". 
here is the error CI test pipeline: [Linux Static CC (Ubuntu 22.04, Python 3.11, Clang)](https://github.com/openvinotoolkit/openvino/actions/runs/8531544197/job/23374316592)